code/src/components/app-loader/AppLoader.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from "@testing-library/react";
+import AppLoader from "@/components/app-loader/AppLoader";
+
+describe("AppLoader", () => {
+  test("renders AppLoader props", () => {
+    render(<AppLoader />);
+    const loader = screen.getByRole("progressbar");
+    expect(loader).toBeInTheDocument();
+  });
+
+  test("renders AppLoader with additional class names", () => {
+    render(<AppLoader className="custom-class" />);
+    const loader = screen.getByRole("progressbar");
+    expect(loader).toBeInTheDocument();
+    expect(loader).toHaveClass("spa-loader");
+    expect(loader).toHaveClass("custom-class");
+  });
```

Comment:
Is is nessesary test? One test is not enough?

Reply:
removed